content/pages/mua-summary.md

```diff
@@ -0,0 +1,68 @@
+Title: MUA Summary
+Status: hidden
+
+Anyone with a user account to a WiCS machine must sign our [Machine Usage Agreement]({filename}/pages/mua.md),
+summarized below. Those with guest accounts do not need to sign the MUA, but they are still held
+to the rules and regulations outlined in it.
+
+# Acceptable and Unacceptable Use #
+
+The WiCS machines are to be used for learning, class work, personal projects, and 
+use in accordance with [WiCS aims]({filename}/pages/about.md), that does not violate
+any of WiCS' usage policies.
+
+If you have a project of interest that does not fall into these categories, you may
+request special permission from wics-sys.
+
+# User Responsibilities #
+
+All usage of WiCS machines is governed by our [CoC]({filename}/pages/code-of-conduct.md), 
+our [MUA]({filename}/pages/mua.md), local laws, and University policy. Users are also
+responsible for being aware of and adhering to policy of machines connected to from 
+WiCS machines. This means, among other things, that machines may not be used for
+commercial purposes, including the hosting of corporate websites and mining cryptocurrency.
+
+WiCS is not responsible for any activity on user accounts. Users will be held accountable 
+for all damaging, unethical and illegal activity.
+
+Users should be considerate of others' needs when using WiCS machines. Do not disrupt the 
+work of anyone else, and do not hog resources.
+
+# Security #
+
+Users are only permitted to access their own accounts. Do not attempt to gain access to another’s
+account or personal file. 
+
+Do not attempt to obtain anyone else's password, and do not give your password or provide 
+access to your account to anyone else. 
+
+We recommend you Choose a password that you do not use anywhere else, and do not give a 
+machine automatic access to your WiCS account.
+
+In the event that you discover a security problem, members of [The Systems Committee](mailto:wics-sys@lists.uwaterloo.ca)
+must be notified immediately. The problem should not be exploited, and it should not be made
+public until it is corrected.
+
+# Auditing and Logging #
+
+The Systems Committee will keep logs on all machines for diagnostic and auditing purposes.
```

Comment:
These logs contain records of user and machine activity, and may be shared with University Computing or local law enforcement as required.